src/module.ts

```diff
@@ -35,6 +36,27 @@ export default defineNuxtModule<SwiperModuleOptions>({
         from: 'swiper/vue'
       }
     ]
+    
+    // Separating Swiper into chunks from Nuxt entry bundle
+    nuxt.options.vite = defu(
```

Comment:
I would recommend using the `import { extendViteConfig } from '@nuxt/kit'`. Since Nuxt Kit already maps the Vite config it would better long term to use that or use the nuxt hook `vite:extendConfig` either one works as of today.

Reply:
Good idea